WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "com_github_nlohmann_json")
```

Comment:
Is this file really required in this repository?

Reply:
Unfortunately. See this error:

```
ERROR: The 'build' command is only supported from within a workspace (below a directory having a WORKSPACE file).
```

Reply:
Then I would rather have the bazel build in a different repository, because this seems rather intrusive. Is this possible?

Reply:
That is probably already covered by the original OP's demo project. Closing this.